Panic on launch after Hyprland update

After applying the latest Hyprland update yofi will no longer launch.

I have duplicated this on two different machines and tried running Hyprland with the default config to rule out an issue with my configs.

Attempting to run from a terminal creates the following output:

```plaintext
[2023-12-16][20:41:06][yofi][WARN] cannot read blacklist file "/home/user/.config/yofi/blacklist": No such file or directory (os error 2)
interface 'wl_surface' has no event 2
thread 'main' panicked at 'called Result::unwrap() on an Err value: OtherError(Os { code: 0, kind: Uncategorized, message: "Success" })', src/main.rs:242:44
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
[2023-12-16][20:41:06][yofi][ERROR] panic: called Result::unwrap() on an Err value: OtherError(Os { code: 0, kind: Uncategorized, message: "Success" })
```

I am on Arch Linux and have tried both the `yofi` and `yofi-git` AUR packages. Hyprland is from the Arch repos.

I have confirmed that yofi still works in sway. So perhaps this is related to a newer build of wlroots, or something else specific to Hyprland.

I would be happy to provide further information if needed.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ fontconfig = { version = "0.5.0", features = ["dlopen"] }
 
 anyhow = "1.0.51"
 wayland-protocols = { version = "0.29.5", default-features = false, features = ["unstable_protocols", "client"] }
-sctk = { version = "0.16.0", package = "smithay-client-toolkit", features = ["calloop"] }
+sctk = { version = "0.16.1", package = "smithay-client-toolkit", features = ["calloop"] }
 raqote = { version = "0.8.1", default-features = false }
 xdg = "2.4.0"
 fep = { version = "1.3.0", package = "freedesktop_entry_parser" }

